hikari/guilds.py

```diff
@@ -246,6 +247,23 @@ class GuildVerificationLevel(int, enums.Enum):
     """Must have a verified phone number."""
 
 
+@typing.final
+class GuildNSFWLevel(int, enums.Enum):
+    """Represents the NSFW level of a guild."""
+
+    DEFAULT = 0
+    """Default level of NSFW (per channel)."""
+
+    EXPLICIT = 1
+    """Guild contains explicit NSFW content."""
+
+    SAFE = 2
+    """Guild is safe of NSFW content."""
```

Comment:
So does this mean that there's no channels marked as nsfw in the guild or that it's not age restricted?

Reply:
At the current time, there is no way of setting this (although the [article](https://support.discord.com/hc/en-us/articles/1500005389362-NSFW-Server-Designation) does mention this will come in the future)

I will rewrite the docstrings for all of them to see if I can make them clear :)